#Request
geometry_msgs/Pose target_pose1
geometry_msgs/Pose target_pose2
string item
uint8 priority
---
#Response
bool accepted
